src/api-docs.js

```diff
@@ -60,6 +60,25 @@ const defaultApiDocs = {
                     type: 'string',
                     description: 'Key of the multisig wallet data in the config. This allow wallets to be started without a seedKey, e.g. with the seed on the parameters or from an xpubkey.',
                   },
+                  scanPolicy: {
+                    type: 'string',
+                    enum: ['gap-limit', 'index-limit'],
+                    description: 'Address scanning policy to use.',
+                    default: 'gap-limit',
+                  },
+                  gapLimit: {
+                    type: 'number',
+                    description: 'Gap limit to use when scanning addresses. Only used when scanPolicy is set to \'gap-limit\'. If not given the configured default will apply.',
+                  },
+                  policyStartIndex: {
+                    type: 'number',
+                    description: 'Load addresses starting from this index. Only used when scanPolicy is set to \'index-limit\'.',
+                    default: 0,
+                  },
+                  policyEndIndex: {
+                    type: 'number',
+                    description: 'Stop loading addresses at this index. Only used when scanPolicy is set to \'index-limit\'. Defaults to policyStartIndex',
+                  },
```

Comment:
[💬 comment] This is interesting, I thought that a `index-limit` policy were a complement to the `gap-limit` policy, like an upper bound limit.

Reply:
Got it, thank you!